docs/examples/advanced_http.py

```diff
@@ -0,0 +1,12 @@
+from nameko.web import http
+
+
+class Service(object):
+
+    @http('GET', '/404')
+    def does_not_exist(self, value):
+        return 404, "Does Not Exist"
+
+    @http('GET', '/google')
+    def redirect(self,):
+        return 302, "", {'Location': 'https://www.google.com/'}
```

Comment:
with 3 headers, it's `status, headers, payload` (probably worth documenting; possibly along with the fact that this is built on top of werkzeug and also accepts a werkzeug response object for complete customisation)